packages/cli/src/lingui-compile.ts

```diff
@@ -90,17 +89,6 @@ export async function command(
           namespace
         )
 
-        if (options.typescript) {
```

Comment:
Could it break something for the existing users?

Reply:
I don't think so. Previously, we were generating a `d.ts` file along with `.ts`. 

Now it'll be only `.ts` and the `d.ts` is safe to be removed. Perhaps that can be mentioned in the release notes.